src/verifier.js

```diff
@@ -147,8 +147,8 @@ function validateClaimType(values, claim, array, type) {
 
 function validateClaimValues(values, claim, allowed, arrayValue) {
   const failureMessage = arrayValue
-    ? `None of ${claim} claim values are allowed.`
-    : `The ${claim} claim value is not allowed.`
+    ? `None of ${claim} claim values are allowed: ${JSON.stringify(values)}`
+    : `The ${claim} claim value is not allowed: ${JSON.stringify(values[0])}`
```

Comment:
as this is a message that's presented to the user, shall we just represent it without serializing it to json?

Reply:
I see two benefits of JSON serialisation:
1. The values are enclosed in double-quotes. Consider the case when the claim is an empty string - do we want the error message to end with `not allowed: ""` or `not allowed: `?
2. Can a claim value be an object? Converting an object to a string returns `[object Object]`.

Having said that, I am ok to rework this to use a simple string concatenation instead of `JSON.stringify`.